src/inputs/Table/Table.js

```diff
@@ -46,16 +47,23 @@ const useStyles = makeStyles((theme) => ({
     marginTop: 'auto',
     marginBottom: 'auto',
   },
-}));
 
-const LOADING_STATUS_MAPPING = {
-  [TABLE_DATA_STATUS.EMPTY]: false,
-  [TABLE_DATA_STATUS.UPLOADING]: true,
-  [TABLE_DATA_STATUS.DOWNLOADING]: true,
-  [TABLE_DATA_STATUS.PARSING]: true,
-  [TABLE_DATA_STATUS.READY]: false,
-  [TABLE_DATA_STATUS.ERROR]: false,
-};
+  noTableTextDiv: {
+    textAlign: 'center',
+    padding: theme.spacing(4, 0),
+    width: '15%',
+    marginRight: 'auto',
+    marginLeft: 'auto',
+  },
+
+  noTableTextTitle: {
+    marginBottom: theme.spacing(1),
+  },
+
+  noTableTextDescription: {
+    marginBottom: theme.spacing(4),
+  },
+}));
```

Comment:
Could you rename the classnames to replace `noTableText` by `emptyTablePlaceholder`?

Reply:
DONE